develop-guide/writeblitz/writeblitz.rst

```diff
@@ -877,6 +912,41 @@ Below you can see the example of regex filter
         bootflash:
           value: "The bootflash is %VARIABLES{bootflash} and %VARIABLES{measure}"
 
+For actions that has list outputs you can get an index or a part of a list and save it into a list with a desired variable_name.
+You can also create a regex filter and match it against all the items within that list, and get a list of 
+all the matched items.
+
+Below you can see the example of list filter.
+
+.. code-block:: YAML
+
+    # saves various items of a list with a variable
+
+    - api:
+        device: PE1
+        function: get_list_items
+        arguments:
+            name: [{'a': 1}, {'d': {'c': 'name1'}}, [1,2,34], {'e': ['a', 'b', 'c']}]
+            index: 0
+            index_end: 5
+        save:
+            - variable_name: list_int5          # the output is [{'a': 1}, {'d': {'c': 'name1'}}, [1,2,34], {'e': ['a', 'b', 'c']}]
+              list_index: "[0:2]"               # saves items 0,1 from the above array of itmes => [{'a': 1}, {'d': {'c': 'name1'}}]
+                                                # into a list named list_int5
+            - variable_name: list_int7
+              list_index: 2                     
+            - variable_name: list_int8          # saves item #2 in the array =>[[1,2,34]] into a list name list_int7
```

Comment:
comment is not on the right line?

And list_int8 doesnt have comment

Reply:
done